### PR TITLE
IDEM-52

### PR DESCRIPTION
**Pull request by JasminFarhi**
Added fields for user's address details
Image from iOS (108)
Image from iOS (107)

### CODE REVIEW

**Review by amagon270 — APPROVED** [review]

File: app/src/utils/claim-utils.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const displayClaimValue = (claim: ClaimWithValue): string => {
     EmailCredential: (value: any) => value.email,
     MobileNumberCredential: (value: any) => value.mobileNumber,
     AddressCredential: (value: any) =>
-      value.propertyNumber + " " + value.streetName + ", " + value.postCode,
+      value.HouseNumber + " " + value.streetName + ", " + value.postCode,
```
Comment (bitcoinbrisbane) [inline app/src/utils/claim-utils.ts:70]: Not to nit pick, but lest use a template

```suggestion
      `${value.HouseNumber} ${value.streetName}, ${value.postCode}`,
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Small change

**Review by jrocca82 — APPROVED** [review]

File: app/src/data/claims.ts [hunk 2]
```diff
@@ -70,12 +70,14 @@ const claims: Claim[] = [
...
     verificationAction: "document-upload",
     fields: [
-      { id: "propertyNumber", title: "Property Number", type: "text" },
-      { id: "streetName", title: "Street Name", type: "text" },
-      { id: "postCode", title: "Post Code", type: "text" }
+      { id: "HouseNumber", title: "Unit/House Number", type: "text" },
+      { id: "streetName", title: "Street address", type: "text" },
+      { id: "suburb", title: "Suburb", type: "text" },
+      { id: "postCode", title: "Post Code", type: "text" },
+      { id: "state", title: "State", type: "text" }
```
Comment (jrocca82) [inline app/src/data/claims.ts:80]: @bitcoinbrisbane do we want to add country in here, make it international?

**Review by bitcoinbrisbane — APPROVED** [review]

File: app/src/data/claims.ts (hunk 2 above)
Comment (bitcoinbrisbane) [reply app/src/data/claims.ts:80]: Yeah, good catch.

**Pull request merged**